src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
@binarin can you please revert the change that makes only one node alarmed? I failed to convince other team members, even though I completely agree it is a more reasonable behaviour. For 3.7 we would consider making this configurable, with a possible new mode: voluntary node shutdown.